src/core/dbus-manager.c

```diff
@@ -1968,13 +1969,16 @@ static int method_enable_unit_files_generic(
         if (r < 0)
                 return r;
 
+        flags = (runtime & UNIT_FILE_RUNTIME) |
+                (force & UNIT_FILE_FORCE);
```

Comment:
given that this expression is used multiple times, maybe we should add add a macro for this, UNIT_FILE_FLAGS_FROM_BOOLS(runtime, force)...

Which we then can use everywhere?